gordo_components/workflow/server_to_sql/server_to_sql.py

```diff
@@ -56,7 +56,10 @@ def server_to_sql(
 
     Parameters
     ----------
-    TODO: FIll in missing params
+    project_name: str
+        Name of the project to run against.
+    server_address:
+        Address of the server to connect to. ie. https://myhost.com
     sql_host: str
         Host where postgres resides, e.g. "localhost"
```

Comment:
e.g. - exempli gratia ‘for the sake of example’ ... used when giving an example.
i.e. - id est ‘that is’ ... i.e. 'to put it another way' or 'specifically the thing I am referring to'

And punktums here and below.

Reply:
Thanks for the grammar lesson.